src/scheduling.py

```diff
@@ -42,24 +43,30 @@ def logout():
 
     @app.route("/login", methods=["POST"])
     def login():
+        """Using POST method for Employee login"""
         email = request.form["email"]
         password = request.form["password"]
         user = database.find_user(email)
+
         if user is None and not database.get_users():
+            # Creates Employee in the database if not exists
             user = database.create_user(
                 email, password, "Admin", hours_limit=0.0, admin=True
             )
         elif user is None:
+            """Redirects the user if not found in the database"""
             return redirect("/#user_not_found")
 
         response = make_response(redirect("/welcome"))
         response.set_cookie(USER_ID_COOKIE, str(user.id), secure=False)
         return response
 
     # Mark: User Actions
+    """Creates Employee in the database using 'POST method' """
 
     @app.route("/create_user", methods=["POST"])
     def create_user():
+        """ " Creates Employee"""
```

Comment:
Let's remove the extra quote (") character before 'Creates'

Reply:
Ok will look into it.